tuf-spec.md

```diff
@@ -1318,15 +1318,34 @@ it in the next step.
   report the potential freeze attack.  On the next update cycle, begin at step
   [[#update-root]] and version N of the root metadata file.
 
-11. **If the timestamp and / or snapshot keys have been rotated, then delete the
-  trusted timestamp and snapshot metadata files.** This is done
-  in order to recover from fast-forward attacks after the repository has been
-  compromised and recovered. A _fast-forward attack_ happens when attackers
-  arbitrarily increase the version numbers of: (1) the timestamp metadata, (2)
-  the snapshot metadata, and / or (3) the targets, or a delegated targets,
-  metadata file in the snapshot metadata. Please see [the Mercury
-  paper](https://theupdateframework.io/papers/prevention-rollback-attacks-atc2017.pdf)
-  for more details.
+11. **Fast-forward attack recovery** A _fast-forward attack_ happens
+  when attackers arbitrarily increase the version numbers in any of the
+  timestamp, snapshot, targets, or delegated targets metadata. The attacker's goal
+  is to cause clients to refuse to update the metadata later because the attacker's
+  listed metadata version number (possibly MAX_INT) is greater than the new valid
+  version.  To recover from a fast-forward attacks after the repository has been
+  compromised and recovered, certain metadata files need to be deleted as
+  specified in this section. If a delegated targets file is subjected to a
+  fast-forward attack, the snapshot role's keys should be replaced. Please see
+  [the Mercury paper](https://ssl.engineering.nyu.edu/papers/kuppusamy-mercury-usenix-2017.pdf)
+  for more details on fast-forward attacks.
+
+    1. **Targets recovery** If a threshold of targets keys have been
+    removed in the new trusted root metadata compared to the previous trusted
+    root metadata, delete the old top-level targets and snapshot metadata
+    files. Note that delegated targets are susceptible to fast forward attacks,
```

Comment:
I have to admit these two sentences are not immediately clear to me. Would you please rephrase and clarify?